xarray/tests/test_backends.py

```diff
@@ -1139,6 +1257,8 @@ def test_dataarray_compute(self):
         self.assertTrue(computed._in_memory)
         self.assertDataArrayAllClose(actual, computed)
 
+class DaskTestAutocloseTrue(DaskTest):
+    autoclose=True
```

Comment:
I'm going to ignore PEP8 to `xarray/core/pycompat.py` because code in there is essentially a copy / paste.